Bottom: d98c19f7964f394e6c61aee9f02935aa92c24a9a
Top:    99ba895e26c951222e77dd2c0a9080b8f73455dd
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 09:17:44 -0500

Update ptxdistrc filename


---
diff --git a/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git b/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
similarity index 100%
rename from docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git
rename to docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
